### PR TITLE
Update the spec to cover the situation when reading from or drawing to a missing attachment

### PR DESCRIPTION
**Pull request by Richard-Yunchao**
Update the spec per #2137. 

PTAL when you have time. Thanks a lot! @zhenyao and @kenrussell .

### CODE REVIEW

**Comment by kenrussell** [comment]
Per #2137 let's examine this after Chrome 56's branch point late this Thursday Pacific time.

**Comment by Richard-Yunchao** [comment]
Now that the Chromium M56 has been checked out. Furthermore, Ken has taken the WebGL 2.0.0 snapshot for WebGL CTS. I suppose that we would have WebGL 2.0.0 specification snapshot soon. After that, new revisions for WebGL 2.0 features (including this one if it is OK) should be added into 2.0.1 spec. Right? @kenrussell , @zhenyao

File: specs/latest/1.0/index.html [hunk 1]
```diff
@@ -2971,6 +2971,12 @@ <h2 class="no-toc">Table of contents</h2>
         of the next compositing operation.
     </p>
 
+    <p>
+        If anyone of these 3 calls attempts to draw to a missing attachment of a complete framebuffer,
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: anyone -> any one

File: specs/latest/1.0/index.html [hunk 2]
```diff
@@ -2971,6 +2971,12 @@ <h2 class="no-toc">Table of contents</h2>
...
+        nothing is to be drawn to that attachment and no error is generated
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: nothing is to be drawn -> nothing is drawn

File: specs/latest/1.0/index.html [hunk 3]
```diff
@@ -3834,6 +3840,23 @@ <h2 class="no-toc">Table of contents</h2>
     </ul>
 </p>
 
+<h3><a name="DRAWING_TO_MISSING_ATTACHMENT">Drawing To a Missing Attachment</a></h3>
+<p>
+    In the OpenGL ES 2.0 API, it is not specified what happens when a command tries to drawing
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: tries to drawing -> tries to draw

File: specs/latest/1.0/index.html [hunk 4]
```diff
@@ -3834,6 +3840,23 @@ <h2 class="no-toc">Table of contents</h2>
...
+    to a missing attachment, such as clearing a draw buffer from a complete framebuffer
+    that does not have a color attachment.
+</p>
+<p>
+    In the WebGL API, such operations that drawing to an attachment that is missing will draw
```
Comment (kenrussell) [inline specs/latest/1.0/index.html]: that drawing -> that draw

File: specs/latest/2.0/index.html [hunk 5]
```diff
@@ -1281,10 +1281,18 @@ <h2 class="no-toc">Table of contents</h2>
...
         framebuffer. If the value of SAMPLE_BUFFERS for the read framebuffer is one and the value of
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
-        the source are converted to a single sample before being written to the destination.
+        the source are converted to a single sample before being written to the destination.<br><br>
 
         For any source pixel lying outside the read framebuffer, the corresponding destination pixel remains
-        untouched.
+        untouched.<br><br>
+
+        If this function attempts to blit to a missing attachment of a complete framebuffer, nothing is to be
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: nothing is to be -> nothing is

File: specs/latest/2.0/index.html [hunk 6]
```diff
@@ -1281,10 +1281,18 @@ <h2 class="no-toc">Table of contents</h2>
...
+        blitted to that attachment and no error is generated
+        per <a href="../1.0/index.html#DRAWING_TO_MISSING_ATTACHMENT">Drawing to a Missing Attachment</a>.<br>
+
+        If this function attempts to read from a missing attachment of a complete framebuffer, and at least one
+        draw buffer has image to be blitted, an <code>INVALID_OPERATION</code> error is generated
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: has image -> has an image

File: specs/latest/2.0/index.html [hunk 7]
```diff
@@ -2308,8 +2320,13 @@ <h2 class="no-toc">Table of contents</h2>
         </table><br>
         If <code>buffer</code> is <code>COLOR_BUFFER</code> and the function is not chosen according to the
         above table, an <code>INVALID_OPERATION</code> error is generated and nothing is cleared.
+
         <p>For ArrayBufferView entrypoints, if there's not enough elements in <code>values</code>
            starting at <code>srcOffset</code>, generate <code>INVALID_VALUE</code>.
+
+        <p>If this function attempts to clear to a missing attachment of a complete framebuffer, nothing
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: clear to a -> clear a

File: specs/latest/2.0/index.html [hunk 8]
```diff
@@ -2308,8 +2320,13 @@ <h2 class="no-toc">Table of contents</h2>
...
+           is to be cleared and no error is generated
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: is to be cleared -> is cleared

**Comment by Richard-Yunchao** [comment]
Thanks for your review, @kenrussell . All grammatical  errors are fixed. 

Please help to merge this one if #2179 is merged (or whenever you think it is appropriate to merge it).

**Comment by jdashg** [comment]
LGTM

**Comment by kenrussell** [comment]
Thanks @Richard-Yunchao and sorry for the holdup. I've prepared the 2.0.0 spec snapshot in #2196 so merging this now.

**Comment by Richard-Yunchao** [comment]
Thanks for merging, @kenrussell .

**Pull request merged**